week-4/D-methods/exercise-4.js

```diff
@@ -7,9 +7,10 @@ Define a method "makeFriend" to add a new friend to her list.
 var person = {
     name: "Alice",
     friends: ["John", "Nina"]
+    makeFriend: function() {
+       this.friends.push("bob");
```

Comment:
Nearly right! In line 18 you're calling the method `person.makeFriend("Bob");` however the argument value isn't used anywhere.